newsroom/wire/search.py

```diff
@@ -46,6 +48,12 @@ def get(self, req, lookup):
             }
         }
 
+        user = get_user()
+        company = get_user_company(user)
+        if company and company.get('services'):
+            services = [code for code, is_active in company['services'].items() if is_active]
+            query['bool']['must'].append({'terms': {'service.code': services}})
```

Comment:
So if user has no company there's no filtering for now.

Reply:
yep exactly, I keep it simple now for testing, but this won't be the case eventually